DOC-813: Document fixed_toolbar_container_target setting added in TinyMCE 5.8 (#1958)
tinymce/tinymce-docs: release-notes/release-notes58.md

### Feature name

### Added new `fixed_toolbar_container_target` setting

> Added a new `fixed_toolbar_container_target` setting which renders the toolbar in the specified `HTMLElement`. Patch contributed by [pvrobays](https://github.com/pvrobays).

Author: The first sentence sounds like you're changing it to use passive voice, so I'd rather find a different way to fix that if you don't like the original. Happy to take the second sentence though, it sounds a lot nicer

> Added a new `fixed_toolbar_container_target` setting which renders the toolbar in the specified `HTMLElement`. Patch contributed by [pvrobays](https://github.com/pvrobays).

Reviewer: It definitely needs changing and should not start with `Added`. How you do that is up to you, but remember it's meant to be more like a marketing spiel to say what's been added and what it does.

> Added a new `fixed_toolbar_container_target` setting which renders the toolbar in the specified `HTMLElement`. Patch contributed by [pvrobays](https://github.com/pvrobays).

Reviewer 2: How about something like

```suggestion
The new `fixed_toolbar_container_target` option allows the toolbar to be rendered in a specified `HTMLElement`. This feature was contributed by [pvrobays](https://github.com/pvrobays).
```

Reviewer: Yeah, this looks good to me and matches what's normally used for release notes. We may also want to mention it's similar to the `fixed_toolbar_container` option here just so people are aware when reading this without going to have to read the full details in the configuration docs.

> For more information on `fixed_toolbar_container_target`, see [User interface options: `fixed_toolbar_container_target`]({{ site.baseurl }}/configure/editor-appearance/#fixed_toolbar_container_target).

Reviewer: "More" is generally discouraged based on previous PRs... we also have template for this if you look at the older release notes, so it should be something like this instead:
```suggestion
For information on the `fixed_toolbar_container_target` option, see [User interface options - `fixed_toolbar_container_target`]({{ site.baseurl }}/configure/editor-appearance/#fixed_toolbar_container_target).
```
